be/CMakeLists.txt

```diff
@@ -402,6 +402,11 @@ if (${WITH_CACHELIB} STREQUAL "ON")
     endforeach()
 endif()
 
+if (${WITH_STARCACHE} STREQUAL "ON")
```

Comment:
what if WITH_STARCACHE=OFF and USE_STAROS=ON, will the build break?

Reply:
> what if WITH_STARCACHE=OFF and USE_STAROS=ON, will the build break?

Update